### PR TITLE
🆙 Upgrade electionguard-python dependency to 1.1.5

### LINKED ISSUE #95: Upgrade electionguard-python to 1.1.5
Update the `electionguard-python` dependency to the latest v1.1.5

### PR DESCRIPTION
**Pull request by rkorsak**
### Issue
Closes #95 

### Description
Updates the dependency, and pins all other dependencies to major or minor versions.

### Testing
All tests still pass.

### Checklist
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] 🤔 **CONSIDER** adding a unit test if your PR resolves an issue.
- [x] ✅ **DO** check open PR's to avoid duplicates.
- [x] ✅ **DO** keep pull requests small so they can be easily reviewed.
- [x] ✅ **DO** build locally before pushing.
- [x] ✅ **DO** make sure tests pass.
- [x] ✅ **DO** make sure any new changes are documented.
- [x] ✅ **DO** make sure not to introduce any compiler warnings.
- [x] ❌**AVOID** breaking the continuous integration build.
- [x] ❌**AVOID** making significant changes to the overall architecture.

💔Thank you!

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -11,16 +11,16 @@ documentation = "https://electionguard-web-api.readthedocs.io/"
...
+fastapi = "~0.61"
+uvicorn = "~0.11"
 
 [tool.poetry.dev-dependencies]
-black = { version = "*", allow-prereleases = true }
-mkdocs = "*"
-mypy = "*"
-pylint = "*"
-pytest = "*"
+black = { version = "^20", allow-prereleases = true }
```
Comment (keithrfung) [inline pyproject.toml:19]: We actually might want to be more specific with this one in particular because the changes in 20 I already had issues with.

**Review by keithrfung — CHANGES_REQUESTED** [review]
Just for more accurately specifying version of `black` to the exact version.

File: pyproject.toml (hunk 1 above)
Comment (rkorsak) [reply pyproject.toml:19]: The caret notation should keep it locked to v20 unless we explicitly upgrade, but will allow minor upgrades.  Does that work for you in this case?

Comment (keithrfung) [reply pyproject.toml:19]: It's minor versions. The change on the last a->b had the files flipping back and forth.

**Review by keithrfung — APPROVED** [review]

**Pull request merged**